utils/cq/daos_pylint.py

```diff
@@ -388,6 +388,10 @@ def parse_file(args, target_file, ftest=False, scons=False, fake_scons=False):
         if scons and msg.symbol != 'consider-using-f-string':
             vals['category'] = 'error'
 
+        # Flag some serious warnings as errors
+        if ftest and msg.symbol in ('condition-evals-to-constant'):
```

Comment:
I know I suggested this myself but you don't need to restrict this to ftest.  The reason I suggested in rather than == is so that we can expand this list to include other checks that should never be permitted to pass.
```suggestion
        if msg.symbol in ('condition-evals-to-constant'):
```